src/vmm/src/cpu_config/templates.rs

```diff
@@ -72,6 +72,15 @@ impl From<&Option<CpuTemplateType>> for StaticCpuTemplate {
     }
 }
 
+impl From<&CpuTemplateType> for StaticCpuTemplate {
```

Comment:
Can this preparatory step be extracted in its own commit? If we leave the actual deprecation separate, it can be served as a clear example when we need to do something similar in the future.

Reply:
Sure! I'll split it into smaller commits!

Reply:
done.